src/lib.rs

```diff
@@ -1,3 +1,5 @@
+pub use firewheel_core as core;
```

Comment:
This addition makes working with the derive macros easier both inside and outside Firewheel proper.

I've adopted the Bevy-style _manifest_ approach, which checks whether the macro is being invoked internally or externally. If we re-export `firewheel_core` here in this way, it makes it much easier for the macro to build the correct path.

In general, we may consider doing this for all of Firewheel's internal crates, opting for glob imports in a `prelude` module instead. This certainly isn't necessary, but it would align nicely with Bevy's convention.

Reply:
Sure, you can do that for all of Firewheel's internal crates if you want.

Reply:
Would you want to do that in this PR, or a different one?

Reply:
It should be easy enough to hit in this one since it's at least partially required.

Now, to be clear, are you open to the `prelude` approach here, or just re-exporting the internal crates as `core`, `nodes`, etc?

Reply:
I unified the re-exporting scheme and added a top-level `prelude` module. If you're happy with this scheme, I'll fix the examples.